lib/screens/add_exercise_screen.dart

```diff
@@ -148,28 +155,51 @@ class _DuplicatesAndVariationsStepContent extends StatelessWidget {
   }
 }
 
-class _ImagesStepContent extends StatelessWidget {
-  final GlobalKey<FormState> _imagesStepFormKey = GlobalKey<FormState>();
+class _ImagesStepContent extends StatefulWidget {
+  @override
+  State<_ImagesStepContent> createState() => _ImagesStepContentState();
+}
 
+class _ImagesStepContentState extends State<_ImagesStepContent> with ExcerciseImagePickerMixin {
+  final GlobalKey<FormState> _imagesStepFormKey = GlobalKey<FormState>();
   @override
   Widget build(BuildContext context) {
-    return Form(
-      key: _imagesStepFormKey,
-      child: Column(
-        children: [
-          AddExerciseTextArea(
-            onChange: (value) => print(value),
-            title: 'Name',
-            isRequired: true,
+    return Column(
+      children: [
+        RichText(
+          text: TextSpan(
+            style: Theme.of(context).textTheme.caption,
+            children: const <TextSpan>[
+              TextSpan(text: 'Images must be compatible with the '),
```

Comment:
Can you mark the strings as translatable? You just need to add the translation keys to `app_en.arb`, the generated files are created on build.

And now that I think about it, I would just make this one sentence, without underscoring the license name

Reply:
ok I'll do that